scripts/build_containers.sh

```diff
@@ -102,6 +102,56 @@ function create_production_containers {
 
 }
 
+function create_release_containers {
+  BRANCH=$1
+
+  # Pull images if available for caching
+  docker pull "${CONTAINER_REPO}"/"${CONTAINER_APP}":builder ||:
```

Comment:
@citizen428 This function is largely copied from [the `create_pr_containers` function on the `main` branch](https://github.com/forem/forem/blob/7a8f854d2c645a5ed62530c58bc8c5b0d34ef9ae/scripts/build_containers.sh#L13-L17) and tweaked for release channels. I don't know if we have any bash scripting conventions and, since it seems to work elsewhere in the script, I didn't mess with it much beyond tagging. 🙂 

The idea was to refactor both of these functions later once we knew more about what we wanted them to do and how building containers for a PR looks different from building release-channel containers. I'm not sure we use PR containers for anything critical at the moment so I'm not sure we have much of a solid foundation for refactoring yet.